src/ai/openvino_model.py

```diff
@@ -0,0 +1,58 @@
+import openvino.properties as props
+from huggingface_hub import snapshot_download
+import openvino as ov
+import torch
+import os
+
+core = ov.Core()
```

Comment:
costants in uppercase?

Reply:
It is not a constant, but a global variable that is common for all classes/methods of that module.